command/connect/envoy/bootstrap_config.go

```diff
@@ -213,19 +213,23 @@ func (c *BootstrapConfig) ConfigureArgs(args *BootstrapTplArgs, omitDeprecatedTa
 	}
 	// Setup prometheus if needed. This MUST happen after the Static*JSON is set above
 	if c.PrometheusBindAddr != "" {
-		if err := c.generateListenerConfig(args, c.PrometheusBindAddr, "envoy_prometheus_metrics", "path", "/metrics", "/stats/prometheus"); err != nil {
+		// if args.PrometheusBackendPort is not an empty string, this will tell
```

Comment:
I feel like this comment belongs somewhere in `generateListenerConfig` instead

Reply:
Sounds good, I removed this and modified the comment in `generateListenerConfig`